### PR TITLE
115 releasenotescreen migrate to service portal

### PR DESCRIPTION
**Pull request by HavardNot**
Migrates release note API call to Service Portal, 
now uses Markdown converted into HTML for the bullet points in the release notes

### CODE REVIEW

**Comment by HavardNot** [comment]
image (5)

File: package.json
Comment (PtjeEquinor) [inline package.json:184]: Moment have vulnerabilities is all versions except the latest. Since we are only using very little of it we could avoid it by using something like this instead

`var options = { year: 'numeric', month: 'short', day: 'numeric' };
var today  = new Date();

console.log(today.toLocaleDateString("en-US", options));`

**Review by PtjeEquinor — APPROVED** [review]
Consider removing moment as its not necessary and earlier versions had vulnerability so this from history its likely this could as well

File: src/components/common/organisms/ChangeLog.tsx
Comment (HaakonSvane) [inline src/components/common/organisms/ChangeLog.tsx:33]: Most of the web apps split large screens into the main view and a backdrop:
Screenshot 2022-12-16 at 10 08 56

I am assuming using the window width could cause issues for these apps?
I had a similar issue some weeks ago and found out that [`<View>`](https://reactnative.dev/docs/view#onlayout) has an `onLayout` prop that can be used to fetch the view dimensions rather than the whole screen dimensions. Maybe this is something that needs to be looked at

**Review by HaakonSvane — APPROVED** [review]
🔥 🥇

File: package.json
Comment (HavardNot) [reply package.json:184]: We can create a Date component in MEC. It is needed and this is a good time to implement it. Setting this issue to blocked until it is resolved

File: src/components/common/organisms/ChangeLog.tsx
Comment (HaakonSvane) [inline src/components/common/organisms/ChangeLog.tsx:29]: Nitpicking, but should this item be typed?
This goes for most of the data we use in MEC i would assume.
It has the benefits of clearly defining a contract that our library needs our api to adhere to. Mapping from API responses to internal library data types also becomes more chill.

**Review by HaakonSvane — APPROVED** [review]
👌

**Review by HaakonSvane — APPROVED** [review]
👍

File: package.json
Comment (HavardNot) [reply package.json]: uninstalled

File: src/components/common/organisms/ChangeLog.tsx
Comment (HavardNot) [reply src/components/common/organisms/ChangeLog.tsx]: changed to use onLayout

File: src/components/common/organisms/ChangeLog.tsx
Comment (HavardNot) [reply src/components/common/organisms/ChangeLog.tsx]: added new type Release with named props

**Review by HaakonSvane — APPROVED** [review]
Good shit 👍

**Comment by HavardNot** [comment]
Pending testing in Notifications Web app before merging to main.
Testing to be conducted by @HaakonSvane

**Comment by HaakonSvane** [comment]
Seems to be working nicely on the web as well 👍

**Pull request merged**
